react-native-google-maps.podspec

```diff
@@ -18,4 +18,5 @@ Pod::Spec.new do |s|
 
   s.dependency 'React'
   s.dependency 'GoogleMaps', '2.5.0'
+  s.dependency 'Google-Maps-iOS-Utils', '2.1.0'
```

Comment:
What do you need it for?

Reply:
I need it to convert the kml tags to Object

Reply:
Ah, I see now :) 
```
 +#import "GMUKMLParser.h"
 +#import "GMUPlacemark.h"
 +#import "GMUPoint.h"
 +#import "GMUGeometryRenderer.h"
```